internal/api/keppel/accounts_test.go

```diff
@@ -1161,21 +1143,6 @@ func TestPutAccountErrorCases(t *testing.T) {
 		ExpectBody:   assert.StringData("no permission for keppel_account:unknown:change\n"),
 	}.Check(t, h)
 
-	// TODO: reenable once we completely remove support for the accounts.in_maintenance flag
```

Comment:
As the comment says, this test is relevant and should be reenabled instead of being deleted.